docker/Dockerfile

```diff
@@ -1,21 +1,58 @@
-FROM pytorch/pytorch:latest
-MAINTAINER nako.sung@navercorp.com
+# Stage - Build
+FROM pytorch/pytorch:1.1.0-cuda10.0-cudnn7.5-runtime as builder
 
-RUN git clone https://github.com/kakao/khaiii.git
-WORKDIR /workspace/khaiii
+WORKDIR /khaiii
 
-RUN pip install cython
-RUN pip install --upgrade pip
+RUN apt-get update -y && \
+    apt-get install -y language-pack-ko
+
+RUN locale-gen en_US.UTF-8 && \
+    update-locale LANG=en_US.UTF-8
+
+COPY requirements.txt .
```

Comment:
@krikit

> Build failed at this point because there is no file.
> Maybe you ommitted cloning from github.

If you build Dockerfile inside `Docker` directory, you must pass parent directory as a context like
`docker build .. -f Dockerfile`

> And most of all, why do you copy files everywhere?

I think updating model might be more occurred than change source of `khaiii`. In this case, with original dockerfile, docker will always rebuild even if original source has not been changed since it copies all files before building `khaiii` and resources. 

It's more efficient to cache build output and reusing them when changing model only, so I splitted copying file task into 2 tasks. As we cannot copy multiple folders with single `COPY` instruction so multiple `COPY` instruction is unavoidable.